lib/commands/star.js

```diff
@@ -0,0 +1,20 @@
+var log = require('loglevel');
+var core = require('../core');
+var h = require('../helper');
+
+var cmd = {
+  command: 'star <keyword>',
+  desc:    'star problem by name or index'
+};
+
+cmd.handler = function(argv) {
+  core.getProblem(argv.keyword, function(e, problem) {
+    if (e) return log.fail(e);
+    core.starProblem(problem, function(e) {
+      if (e) return log.fail(e);
+      log.info(h.prettyText(' Starred', true));
```

Comment:
OK, I think that is good enough for the moment. I guess there will be more patches to address related functions like 'unstar'. Also want to update local cache whenever is_favor is updated in problem json.